.gitlab/pascal/pipeline_compiler_tests.yml

```diff
@@ -123,46 +127,25 @@ release allocation:
     - export JOB_ID=$(squeue -h -n "${JOB_NAME}" -o "%A")
     - ([[ -n "${JOB_ID}" ]] && scancel ${JOB_ID})
 
-# Load the spack shell integration and load the environment.
-.uses spack environment:
-  before_script:
-    - source ${HOME}/${SPACK_REPO}/share/spack/setup-env.sh
-    - source spack-ci-env-name.sh
-    - spack env activate lbann-${SPACK_ENV_NAME}-${SPACK_ARCH_TARGET}
-    - spack load lbann@${SPACK_ENV_NAME}-${SPACK_ARCH_TARGET} arch=${SPACK_ARCH}
-
 # Variables for Pascal.
 .pascal common:
   variables:
     # Just the obvious identifier. Which specific node doesn't matter.
     SYSTEM_NAME: pascal
-    SPACK_REPO: spack_repos/spack_${SYSTEM_NAME}.git
+#    SPACK_REPO: spack_repos/spack_${SYSTEM_NAME}.git
```

Comment:
same

Reply:
Once testing is done these will be reinstated to use system specific git repos.